packages/core/src/util/token.spec.ts

```diff
@@ -230,18 +274,18 @@ describe("getWebidFromTokenPayload", () => {
         "https://some.issuer",
         "https://some.clientId",
       ),
-    ).resolves.toBe("https://some.webid#me");
+    ).rejects.toThrow();
   });
 
-  it("returns the WebID it the 'sub' claim exists and it is IRI-like", async () => {
+  it("throws if the 'webid' claim exists and but 'azp' does not exist", async () => {
```

Comment:
According to https://openid.net/specs/openid-connect-core-1_0.html#IDToken, the `azp` claim is optional, so we don't want to throw even if it is absent.